runtime/src/main/cpp/Exceptions.cpp

```diff
@@ -125,14 +125,14 @@ NO_INLINE OBJ_GETTER0(Kotlin_getCurrentStackTrace) {
   RETURN_OBJ(result.obj());
 #else
   const int maxSize = 32;
-  void* buffer[maxSize];
```

Comment:
Agree, but this line haven't been changed, so I'd better prefer to not beautify it. Or it may be separate commit. I will check indents that I may introduce incorrectly as a result of copying from another source, thanks.